libs/explorers/services/src/lib/platform.service.ts

```diff
@@ -0,0 +1,17 @@
+import { isPlatformBrowser, isPlatformServer } from '@angular/common';
+import { Injectable, inject, PLATFORM_ID } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class PlatformService {
+  readonly platformId: Record<string, any> = inject(PLATFORM_ID);
+
+  public get isBrowser(): boolean {
```

Comment:
isBrowser() and isServer() are essentially wrappers that should be easier to intuitively understand.